packages/thread-context/tests/bridge-test.js

```
@@ -0,0 +1,124 @@
+
+  const client = new BridgeClient(ctx.port, { callTimeout: 5000 });
+  const methodProxy = createMethodProxy(client);
+
+  const result = await methodProxy.callAsync('threadContext.bridge.echo', 'hi');
+  test.equal(result.val, 'hi');
+  test.equal(result.transformed, true);
+
+  ctx.destroy();
+});
+
+Tinytest.addAsync('thread-context - bridge - destroy rejects in-flight calls', async function (test) {
```

ESLint

[error] Replace `Tinytest.addAsync('thread-context·-·bridge·-·destroy·rejects·in-flight·calls',` with `··Tinytest.addAsync(⏎····'thread-context·-·bridge·-·destroy·rejects·in-flight·calls',⏎···`

(prettier/prettier)